tool/speed.cc

```diff
@@ -627,10 +635,18 @@ static bool SpeedAES256XTS(const std::string &name, //const size_t in_len,
 
   return true;
 }
+#endif
 
 static bool SpeedHashChunk(const EVP_MD *md, std::string name,
                            size_t chunk_len) {
+  // OpenSSL 1.0 doesn't use ptr types for EVP_MD_CTX
```

Comment:
Could you elaborate? Not sure what is meant by
> doesn't use ptr types for EVP_MD_CTX

Is `EVP_MD_CTX_create()` what you are looking for?

Reply:
I swear I looked for this and couldn't find it, but you're right